src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    private static final Random random = new Random();
```

Comment:
Avoid using static variables unless they are constants. Here, 'random' should not be static because it's an instance variable that is used to generate random numbers and is not a constant value.